runtime/lua/vim/lsp/_watchfiles.lua

```diff
@@ -107,6 +107,13 @@ local to_lsp_change_type = {
   [watch.FileChangeType.Deleted] = protocol.FileChangeType.Deleted,
 }
 
+--- Default excludes the same as VSCode's `files.watcherExclude` setting.
+--- https://github.com/microsoft/vscode/blob/eef30e7165e19b33daa1e15e92fa34ff4a5df0d3/src/vs/workbench/contrib/files/browser/files.contribution.ts#L261
+---@type Lpeg pattern
+M._poll_exclude_pattern = parse('**/.git/{objects,subtree-cache}/**')
```

Comment:
@folke I wonder if exposing a variable like this would make it possible to exclude additional patterns per language, like `target` for rust.

Reply:
I think it's okay to expose this while keeping it private for us to be able to quickly test out stuff, but if we realize that this is useful to customize more often and before this starts popping up as recommendation to uers we should instead add a new setting for the client.